_content/monographs/guide-to-japanese-buddhism_japan-buddhist-federation.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "A Guide to Japanese Buddhism"
+authors:
+  - "Japan Buddhist Federation"
+editor: "Kōdō Matsunami"
+external_url: "https://library.sakyauniversity.com/themes/default/resources/js/pdfjs/web/viewer.html?file=/eBooks/2023/June/64896eefafa11/guidejapanbuddhismbm6.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/13KKhdZYiJVw5xOzEA8fr2p-jcBinDPKn/view?usp=sharing"
+course: japanese
+tags:
+  - east-asia
+year: 2004
+publisher: "Japan Buddhist Federation"
```

Comment:
I don't like that this is duplicated info. Either move the editor to authors or remove the publisher and move this to booklets